sdk/src/logs/batch_log_record_processor.cc

```diff
@@ -79,21 +79,25 @@ bool BatchLogRecordProcessor::ForceFlush(std::chrono::microseconds timeout) noex
   // Now wait for the worker thread to signal back from the Export method
   std::unique_lock<std::mutex> lk_cv(synchronization_data_->force_flush_cv_m);
 
-  synchronization_data_->is_force_flush_pending.store(true, std::memory_order_release);
+  std::uint64_t current_sequence =
+      synchronization_data_->force_flush_pending_sequence.fetch_add(1, std::memory_order_release) +
```

Comment:
Is the fetch_add here just the same as `synchronization_data_->force_flush_pending_sequence++`?

Reply:
In my understanding, `synchronization_data_->force_flush_pending_sequence++` use `std::memory_order_seq_cst` , we can use `std::memory_order_release` here.